paddle/inference/io.cc

```diff
@@ -60,15 +63,22 @@ void LoadPersistables(framework::Executor& executor,
       new_var->SetLoDLevel(var->GetLoDLevel());
       new_var->SetPersistable(true);
 
-      // append_op
-      framework::OpDesc* op = load_block->AppendOp();
-      op->SetType("load");
-      op->SetOutput("Out", {new_var->Name()});
-      op->SetAttr("file_path", {dirname + "/" + new_var->Name()});
-      op->CheckAttrs();
+      paramlist.push_back(new_var->Name());
     }
   }
+  
+  //sort paramlist to have consistent ordering 
+  std::sort(paramlist.begin(), paramlist.end());
+
+  // append_op
+  framework::OpDesc* op = load_block->AppendOp();
+  op->SetType("load_combine");
+  op->SetOutput("Out", paramlist);
+  op->SetAttr("file_path", {dirname + "/" + "modelparams.save"});
```

Comment:
I think we need the change the input arguments of `Load()`, user should be allowed to specify the filename of program and parameters. We should define another `Load()`.

Reply:
Can you please look at #8024 ? We can close this PR.

Reply:
@sidgoyal78 Yes, we can close this PR.